src/main/java/ddingdong/ddingdongBE/domain/scorehistory/controller/dto/response/ScoreHistoryListResponse.java

```diff
@@ -15,16 +15,16 @@
         description = "어드민 - 동아리 점수 변동 내역 목록 응답"
 )
 @Builder
-public record ScoreHistoryFilterByClubResponse(
+public record ScoreHistoryListResponse(
 
         @Schema(description = "동아리 총 점수", example = "50")
         BigDecimal totalScore,
         @ArraySchema(schema = @Schema(description = "점수내역 목록", implementation = ScoreHistoryResponse.class))
         List<ScoreHistoryResponse> scoreHistories
 ) {
 
-    public static ScoreHistoryFilterByClubResponse of(Club club, List<ScoreHistoryResponse> scoreHistories) {
-        return ScoreHistoryFilterByClubResponse.builder()
+    public static ScoreHistoryListResponse of(Club club, List<ScoreHistoryResponse> scoreHistories) {
```

Comment:
```suggestion
    public static ScoreHistoryListResponse of(
    Club club, 
    List<ScoreHistoryResponse> scoreHistories
    ) {
```
p4) 큰건 아니지만! 그때 얘기 했던 방법대로 바꾸는 게 좋을 것 같기도 합니다..!